packages/ui/components/dao/DaoCard.tsx

```diff
@@ -1,5 +1,129 @@
+        <div
+          className={clsx(
+            'flex absolute top-0 left-0 flex-row items-center p-3 w-full',
+            {
+              'justify-between': !!subDaoInfo,
+              'justify-end': !subDaoInfo, // Keep the pin and member check at the end if no subDao info.
+            }
+          )}
+        >
+          {subDaoInfo && (
+            <Link href={subDaoInfo.parentDaoHref} passHref>
+              <a title={t('info.gotoParent')}>
+                <SubDaoArrow className="fill-icon-interactive-disabled" />
+              </a>
+            </Link>
+          )}
+          <div className="flex flex-row gap-3">
+            <button
+              onClick={onPin}
+              title={pinned ? t('info.pinned') : t('info.pin')}
+            >
+              <DaoCardPin
+                className={clsx('w-4 h-4', {
+                  'fill-icon-secondary': !pinned,
+                  'fill-icon-interactive-active': pinned,
+                })}
+              />
+            </button>
+            <div title={t('info.youAreMember')}>
+              <DaoCardMemberCheck className="w-4 h-4 fill-icon-secondary" />
+            </div>
+          </div>
+        </div>
+
+        <div className="flex flex-col items-center">
+          <div className="relative p-1 rounded-full border-2 border-border-primary">
+            <div
+              className="w-[72px] h-[72px] bg-center bg-cover"
+              style={{
+                backgroundImage: `url(${imageUrl})`,
+              }}
+            ></div>
+            {subDaoInfo && (
+              <Link href={subDaoInfo.parentDaoHref} passHref>
+                <a
+                  className="absolute right-0 bottom-0 w-8 h-8 bg-center bg-cover rounded-full drop-shadow"
+                  style={{
+                    backgroundImage: `url(${subDaoInfo.parentDaoImageUrl})`,
+                  }}
+                ></a>
+              </Link>
+            )}
+          </div>
+          <p className="mt-2 primary-text">{name}</p>
+          <p className="mt-1 caption-text">
+            {new Intl.DateTimeFormat('default', {
+              month: 'long',
+              day: undefined,
+              year: 'numeric',
+            }).format(established)}
+          </p>
+        </div>
+        <div className="w-full">
+          <p className="mb-5 w-full break-words line-clamp-3 secondary-text">
+            {description}
+          </p>
+          <div className="flex flex-row gap-3 items-center mb-2 font-mono caption-text">
+            <Governance className="w-3 h-3" />
+            <p>{t('format.juno', { val: junoBalance })}</p>
+          </div>
+          <div className="flex flex-row gap-3 items-center font-mono caption-text">
+            <DaoCardProposals className="w-3 h-3 fill-current" />
```

Comment:
```suggestion
            <DaoCardProposals className="w-3 h-3 text-current" />
```

Rewrite icon to use `currentColor` internally and change this to use text color classes.